lib/Service/PageService.php

```diff
@@ -389,14 +389,19 @@ private function getIndexPageFile(Folder $folder): File {
 	 * @throws NotFoundException
 	 * @throws NotPermittedException
 	 */
-	public function getPagesFromFolder(int $collectiveId, Folder $folder, string $userId, bool $recurse = false): array {
+	public function getPagesFromFolder(int $collectiveId, Folder $folder, string $userId, bool $recurse = false, bool $forceIndex = false): array {
 		$pageInfos = [];
 		$indexPage = null;
 
 		// Add subpages and recurse over subfolders
 		foreach ($folder->getDirectoryListing() as $node) {
 			if ($node instanceof File && NodeHelper::isPage($node)) {
-				$page = $this->getPageByFile($node, $folder);
+				try {
+					$page = $this->getPageByFile($node, $folder);
+				} catch (NotFoundException) {
+					// If parent folder doesn't have an index page, it throws NotFoundException. Let's ignore it.
```

Comment:
Should the NotFoundExceptions then removed from the doc block?

Reply:
Other code lines in this function can still throw `NotFoundException`.